test-cases/longevity/longevity-large-partition-3h.yaml

```diff
@@ -45,3 +45,6 @@ space_node_threshold: 644245094
 run_fullscan: '{"ks_cf": "scylla_bench.test", "interval": 10}' # 'ks.cf|random, interval(min)'
 
 run_full_partition_scan: '{"ks_cf": "scylla_bench.test", "interval": 2, "pk_name":"pk", "rows_count": 5000, "validate_data": "false"}' # 'ks.cf, interval(sec), partition-key name, number-of-rows-per-partition, validate reversed query output'
+
+# Temporarily downgrade scylla_bench to a stable version
+scylla_bench_version: v0.1.3
```

Comment:
I don't think it happened here as well.

Reply:
right, only in this large-partitions longevity, no s-b issues found.